### DISCUSSION TITLE
how to plug reactor-plc to monitors

### DISCUSSION THREAD

**Discussion by PiasekTheProgramer** [MikaylaFischler/cc-mek-scada#521] [Support]
and for next video you can show how to make more dificult things, and how to make computer be more far away

**Comment by MikaylaFischler** [comment]
Reactor PLC does not support connecting monitors, but that feature was requested once a long time ago, so I may add that at some point. I don't follow what you mean by make computer be more far away

**Comment by MikaylaFischler** [comment]
#526
